src/main/distrib/data/defaults.properties

```diff
@@ -1203,6 +1203,18 @@ web.showGitDaemonUrls = true
 # SINCE 1.7.0
 web.showSshDaemonUrls = true
 
+# Should internal permissions be used for web.otherUrls (external access).
+# If enabled, a user will be offered actions for each "otherUrl" based
+# on the internally managed permissions.
+#
+# Configure with caution: Note that gitblit has no way of knowing if further
+# restrictions are imposed by an external forwarding agent, so this may cause
+# more rights to be advertised than are available. It will not grant additional
+# rights, but may incorrectly offer actions that are unavailable.
+#
+# SINCE 1.7.0
+web.useInternalPermissionsForOtherUrls = false
```

Comment:
Would this be a better name? `web.advertiseAccessPermissionForOtherUrls` and then describe that the configured push transports determine the advertised url permission?